software-setup/software-installation.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Software setup instructions
+---
+
+**Table of contents**
+
+* TOC goes here
+{:toc}
+
+## RStudio Server
+
+You will **not** need to install R or RStudio on your computer.
+Instead, we have set up an RStudio Server for this workshop which you can access entirely via browser.
+Please refer to [these instructions](rstudio-login.md) for setting up your RStudio Server account.
+
+{% if site.workshop_type == "remote" %}
+
+## Zoom setup instructions
+
+We will use Zoom for all remote workshop sessions, as described in our [Workshop Logistics](../workshop/workshop-logistics.md) and [Zoom Procedures](zoom-procedures.md) pages.
+### Installing Zoom
+
+If you do not have Zoom installed yet, you will need to download the Desktop client from <https://zoom.us/download>.
+
+### Enabling screen share
+
+One way that we'll use Zoom is via screen-sharing; it is important that you are able to share your screen with us so we can help you, as needed in 1:1 consulations sessions or in breakout rooms.
+
+If you are working on a Mac, you will need to grant Zoom permissions to be able to use screen share, as follows:
+
+{% comment %}
+Zoom's instructions are not yet updated for Ventura, but we could write instead:
+
+Follow [this link](https://support.zoom.us/hc/en-us/articles/360016688031) for full instructions on how to grant Zoom these permissions, if you have not previously done so.
+{% endcomment %}
```

Comment:
Reviewers, have a look here. I was going to just include Zoom's own docs at first, but they aren't up-to-date, so I added the small subsections below.
Given that we do setup meetings for remote though, I think we can probably just link to Zoom's docs and get rid of the step-by-step instruction below. Thoughts?